### PR TITLE
[CN-31] Update instructions for RHM and OperatorHub users

### PR DESCRIPTION
**Pull request by hasancelik**

### CODE REVIEW

File: .github/operator-release-files/operatorhub-bundle/hazelcast-enterprise-operator.vOPERATOR_VERSION.clusterserviceversion.yaml [hunk 1]
```diff
@@ -44,13 +44,11 @@ spec:
 
     ### Before Your Start
 
-    Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
-
-        $ oc create secret generic hz-license-key-secret --from-literal=key=LICENSE-KEY-HERE
```
Comment (leszko) [inline .github/operator-release-files/operatorhub-bundle/hazelcast-enterprise-operator.vOPERATOR_VERSION.clusterserviceversion.yaml:49]: Isn't that line needed any more?

**Review by leszko — APPROVED** [review]
Thanks for the PR @hasancelik

Added one comment, other than that, LGTM 👍

File: .github/operator-release-files/operatorhub-bundle/hazelcast-enterprise-operator.vOPERATOR_VERSION.clusterserviceversion.yaml (hunk 1 above)
Comment (hasancelik) [reply .github/operator-release-files/operatorhub-bundle/hazelcast-enterprise-operator.vOPERATOR_VERSION.clusterserviceversion.yaml:49]: Thanks for the review 👍 

I have already added same informations/directions into RHM/OperatorHub instructions based on RedHat guys' advise so I removed that part from dashboard.

**Review by mtyazici — APPROVED** [review]
:+1:

**Pull request merged**